questions.md

```diff
@@ -0,0 +1,4 @@
+#Questions
+
+* What's the best way to help students remember all the commandline syntax? 
+* Is it reasonable to start out 1st year, 1st semester web and graphic design students on using terminl and git? 
```

Comment:
YES! The sooner the better. They might not understand "why" they use certain commands and will memorize the sequence of commands, but that early exposure is critical to build the foundation to longterm understanding.